docs/docs/installation.md

```diff
@@ -0,0 +1,68 @@
+---
+# Page settings
+layout: default
+keywords: probabilistic getting started keanu
+comments: false
+permalink: /docs/installation/
+
+# Hero section
+title: Installation
+description: There is an example project that can quickly get your new modelling project off the ground.
```

Comment:
Where's this example project? Is it the same thing as the starter project mentioned later?

Reply:
I think you may be right - is this something that was already on the docs?